seth-shaw-unlv/controlled_access_terms
# [DEP/SEC] Update codesniffer.

**GitHub Issue**: Dependabot alert

# What does this Pull Request do?

Updates composer.json to set codesniffer (dev) requirement to a non-vulnerable version.

# What's new?

- require at least 2.8.1
- don't lock to individual version, allow upgrades within 2.x

# How should this be tested?

Does composer install --require-dev work?
(it should install codesniffer locally)

# Additional Notes:


# Interested parties
@Islandora/committers
